### ISSUE TITLE
Cracklib bug

### ISSUE THREAD

**Issue by antoinerzad1** [swizzin/swizzin#637]
## Bug description
I got a problem when i install swizzin on debian 10 on a oneprovider server.
Cracklib dosent work i have try to install this tool by apt get but he dosent work
![image](https://user-images.githubusercontent.com/40497631/115115586-9a252080-9f95-11eb-8255-a843ca5734df.png)
Can you help me thanks !
## Reproduction steps
1. Install package ' bash <(curl -sL git.io/swizzin) && . ~/.bashrc '

## Installed applications through swizzin
any
## Expected behavior
(A clear and concise description of what you expected to happen.)

## Server Info
**(please complete the following information):**
 - OS: Debian
 - Arch : x86_64
 - Image Source: 

## Is Swizzin up to date?
(Please run `box update`, and report whether that was succesful. Please try to reproduce your issue) again. mention the commit hash you are seeing)

**Comment by userdocs** [comment]
You you have any double or single quotes in your password?

`"` or `'` or `/`

Since it works fine for me.

Debian 10 amd64

![image](https://user-images.githubusercontent.com/16525024/115116403-37318a80-9f91-11eb-88a6-cc637db6d146.png)
![image](https://user-images.githubusercontent.com/16525024/115116449-6cd67380-9f91-11eb-8727-bc8f752a9ad5.png)

**Comment by antoinerzad1** [comment]
No its a password autogenrated by the system

**Comment by flying-sausages** [comment]
Can you please give us more info on what sort of install you're running and where you're getting your hardware from? This seems to be image/vendor specific. If we do not know how to reproduce this, we cannot be of much help.

**Comment by azk1** [comment]
@antoinerzad1 I had the same problem when trying to install with: `bash <(curl -sL git.io/swizzin) && . ~/.bashrc`

```
...     Checking password strength
/etc/swizzin/sources/functions/users: line 113: cracklib-check: command not found
```

Solution:

`export PATH=$PATH:/usr/sbin`
`source ~/.bashrc`

My server is also from oneprovider and I am using their debian 10 template

**Comment by userdocs** [comment]
Here is the default `$PATH` on Debian 10 stock image and Docker.

```
root@server:~# echo $PATH
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

**Comment by liaralabs** [comment]
Any testers for this commit aimed at fixing this issue? I do not have a machine at OneProvider so I cannot re-create the test env directly. I could alter the path before setup on my own machine but this is just going off the assumption that the data @azk1 provided is the only issue at play here on the OneProvider template.

**Comment by liaralabs** [comment]
@azk1 Can you provide any insight as to whether or not you logged in directly via root or `sudo`'d up to root using `sudo bash` or similar?

**Comment by flying-sausages** [comment]
Closing for inactivity, as far as I can tell this has been fixed in #637
